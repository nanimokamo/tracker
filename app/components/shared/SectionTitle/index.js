import React from 'react';

const SectionTitle = ({ children }) => <div className="SectionTitle">{children}</div>;

SectionTitle.propTypes = {
  children: React.PropTypes.node,
};

export default SectionTitle;
