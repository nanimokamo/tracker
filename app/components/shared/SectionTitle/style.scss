.SectionTitle {
  font-size: 12px;
  font-weight: 600;
  color: $MEDIUM_GREY;//GREY_2;
  padding: $PADDING_NORMAL * 0.75 $PADDING_NORMAL;
}
