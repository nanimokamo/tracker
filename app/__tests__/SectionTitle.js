import React from 'react';
import SectionTitle from '../components/shared/SectionTitle';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <SectionTitle>Section title</SectionTitle>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});