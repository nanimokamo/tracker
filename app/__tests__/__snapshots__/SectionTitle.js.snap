exports[`test renders correctly 1`] = `
<div
  className="SectionTitle">
  Section title
</div>
`;
